src/librustc_yk_sections/emit_sir.rs

```diff
@@ -178,11 +186,19 @@ impl<'a, 'tcx, 'gcx> ConvCx<'a, 'tcx, 'gcx> {
                     ret_bb: ret_bb,
                 })
             },
-            TerminatorKind::Assert{target: target_bb, ref cond, ..} =>
+            TerminatorKind::Assert{target: target_bb, ref cond, expected, ..} => {
+                let lcond =  self.lower_operand(cond)?;
+                let local = match lcond {
```

Comment:
Can remove the line above by making this `match self.lower_operand(cond)?`? Doesn't seem any more complex to me!

Reply:
Ah yes.